[std] Failed to build libpcap due to bison package

I recently tried to package the library pcap from [tcpdump](https://www.tcpdump.org). See beta project below:

```ts
import * as std from "std";

export const project = {
  name: "libpcap",
  version: "1.10.4",
};

const source = std
  .download({
    url: `https://www.tcpdump.org/release/libpcap-${project.version}.tar.gz`,
    hash: std.sha256Hash(
      "ed19a0383fad72e3ad435fd239d7cd80d64916b87269550159d20e47160ebe5f",
    ),
  })
  .unarchive("tar", "gzip")
  .peel();

export default (): std.Recipe<std.Directory> => {
  const libpcap = std.runBash`
    ./configure --prefix=/
    make install DESTDIR="$BRIOCHE_OUTPUT"
  `
    .workDir(source)
    .dependencies(std.toolchain())
    .toDirectory();

  return std.setEnv(libpcap, {
    CPATH: { path: "include" },
    LIBRARY_PATH: { path: "lib" },
    PKG_CONFIG_PATH: { path: "lib/pkgconfig" },
  });
};
```

But I encountered this error from the tool bison due to not finding the `m4sugar.m4`.

```bash
ERROR build:bake:bake_inner:run_bake: brioche_core::bake: error=process failed, view full output from these paths:
- /home/container/.local/share/brioche/process-temp/01J3EYTABCGJNTKGB0SGRPB6N0/stdout.log
- /home/container/.local/share/brioche/process-temp/01J3EYTABCGJNTKGB0SGRPB6N0/stderr.log: process exited with status code exit status: 2 scope=Project { project_hash: ProjectHash(Hash("449cd64176ce83927178ae3e2df5d0beedf58fbbc5872b66c23d7ae9d27ce4aa")), export: "default" } recipe_hash=cee9d02ab323ee82c330349d21f4153209639f193372b9139898bdc4aa5eaf02 recipe_kind=Insert recipe_hash=cee9d02ab323ee82c330349d21f4153209639f193372b9139898bdc4aa5eaf02 recipe_kind=Insert
[175] config.status: creating grammar.y                                                                                                                                                                                                                     
[175] config.status: creating pcap-filter.manmisc                                                                                                                                                                                                           
[175] config.status: creating pcap-linktype.manmisc                                                                                                                                                                                                         
[175] config.status: creating pcap-tstamp.manmisc                                                                                                                                                                                                           
[175] config.status: creating pcap-savefile.manfile                                                                                                                                                                                                         
[175] config.status: creating pcap.3pcap                                                                                                                                                                                                                    
[175] config.status: creating pcap_compile.3pcap                                                                                                                                                                                                            
[175] config.status: creating pcap_datalink.3pcap                                                                                                                                                                                                           
[175] config.status: creating pcap_dump_open.3pcap                                                                                                                                                                                                          
[175] config.status: creating pcap_get_tstamp_precision.3pcap                                                                                                                                                                                               
[175] config.status: creating pcap_list_datalinks.3pcap                                                                                                                                                                                                     
[175] config.status: creating pcap_list_tstamp_types.3pcap                                                                                                                                                                                                  
[175] config.status: creating pcap_open_dead.3pcap                                                                                                                                                                                                          
[175] config.status: creating pcap_open_offline.3pcap                                                                                                                                                                                                       
[175] config.status: creating pcap_set_immediate_mode.3pcap                                                                                                                                                                                                 
[175] config.status: creating pcap_set_tstamp_precision.3pcap                                                                                                                                                                                               
[175] config.status: creating pcap_set_tstamp_type.3pcap                                                                                                                                                                                                    
[175] config.status: creating rpcapd/Makefile                                                                                                                                                                                                               
[175] config.status: creating rpcapd/rpcapd.manadmin                                                                                                                                                                                                        
[175] config.status: creating rpcapd/rpcapd-config.manfile                                                                                                                                                                                                  
[175] config.status: creating testprogs/Makefile                                                                                                                                                                                                            
[175] config.status: creating config.h                                                                                                                                                                                                                      
[175] config.status: executing default-1 commands                                                                                                                                                                                                           
[175] gcc -fvisibility=hidden  -fpic -I.  -DBUILDING_PCAP -Dpcap_EXPORTS -DHAVE_CONFIG_H  -g -O2    -c ./pcap-linux.c                                                                                                                                       
[175] gcc -fvisibility=hidden  -fpic -I.  -DBUILDING_PCAP -Dpcap_EXPORTS -DHAVE_CONFIG_H  -g -O2    -c ./fad-getad.c                                                                                                                                        
[175] gcc -fvisibility=hidden  -fpic -I.  -DBUILDING_PCAP -Dpcap_EXPORTS -DHAVE_CONFIG_H  -g -O2    -c ./pcap-usb-linux.c                                                                                                                                   
[175] gcc -fvisibility=hidden  -fpic -I.  -DBUILDING_PCAP -Dpcap_EXPORTS -DHAVE_CONFIG_H  -g -O2    -c ./pcap-netfilter-linux.c                                                                                                                             
[175] gcc -fvisibility=hidden  -fpic -I.  -DBUILDING_PCAP -Dpcap_EXPORTS -DHAVE_CONFIG_H  -g -O2    -c ./pcap.c                                                                                                                                             
[175] bison -p pcap_ -o grammar.c -d grammar.y                                                                                                                                                                                                              
[175] bison: //share/bison/m4sugar/m4sugar.m4: cannot open: No such file or directory                                                                                                                                                                       
[175] make: *** [Makefile:547: grammar.c] Error 1                                                                                                                                                                                                           
Process 175 [1:47.2 Exited 2]
[100%] Downloaded https://www.tcpdump.org/release/libpcap-1.10.4.tar.gz
[100%] Unarchived
Error: process failed, view full output from these paths:
- /home/container/.local/share/brioche/process-temp/01J3EYTABCGJNTKGB0SGRPB6N0/stdout.log
- /home/container/.local/share/brioche/process-temp/01J3EYTABCGJNTKGB0SGRPB6N0/stderr.log: process exited with status code exit status: 2
```

This file does exist by looking at the output of the project `std`:

```bash
bash-5.2$ brioche build -p packages/std/ -o /tmp/std        
Build finished, completed (no new jobs) in 1.76s
Result: 2b5f3069c52f99b08c3bccf9a5d48a8a17c64747c0e72d2ff13e78c512800964
Writing output
Wrote output to /tmp/std

bash-5.2$ find /tmp/std -name m4sugar.m4         
/tmp/std/share/bison/m4sugar/m4sugar.m4
/tmp/std/share/autoconf/m4sugar/m4sugar.m4
```

I'm suspecting this path `//share/bison/m4sugar/m4sugar.m4` with the double slash.

--- a/packages/std/toolchain/native/index.bri
+++ b/packages/std/toolchain/native/index.bri
@@ -210,6 +210,9 @@ export const toolchain = std.memo(
       // pkg-config search paths
       PKG_CONFIG_PATH: { append: [{ path: "lib/pkgconfig" }] },
 
+      // Bison data dir
+      BISON_PKGDATADIR: { fallback: { path: "share/bison" } },
+
       // Magic patterns used by `file`
       MAGIC: { append: [{ path: "share/misc/magic.mgc" }] },
 

